pkg/config/setup/otlp.go

```diff
@@ -66,8 +66,8 @@ func OTLP(config pkgconfigmodel.Setup) {
 	// Metrics settings
 	config.BindEnv("otlp_config.metrics.tags") // TODO OTLP team: add default value
 	config.BindEnvAndSetDefault("otlp_config.metrics.enabled", true)
-	config.BindEnv("otlp_config.metrics.resource_attributes_as_tags")             // TODO OTLP team: add default value
-	config.BindEnv("otlp_config.metrics.instrumentation_scope_metadata_as_tags")  // TODO OTLP team: add default value
+	config.BindEnv("otlp_config.metrics.resource_attributes_as_tags") // TODO OTLP team: add default value
+	config.BindEnvAndSetDefault("otlp_config.metrics.instrumentation_scope_metadata_as_tags", true)
```

Comment:
Looks like you are changing the default for OTLP ingestion here - which is not covered in https://github.com/DataDog/datadog-agent/pull/36940. In this case I'd prefer to still keep your PR but dedicated to this change + release notes